stx-genesis/chainstate.txt

```diff
@@ -0,0 +1,32 @@
+### DEMO / TEST CHAINSTATE DATA
+
+-----BEGIN STX BALANCES-----
+address,balance
+31xUrDNFibogSp72hkpP9NQhns9YhekLLv,347222222225
+322XM9xWxso2AV4d7P3D4JFAJd5UjjxVpa,69444444445
+329V64qvhqxpdP2idN93tY3TbE9qasVVEH,347222222225
+32E6hsvSf5TP2TWSBKkccHcDtF5Uk9Mnki,347222222225
+32FCnf9V7CmuP58BNKW6taybVDjpPh9vLx,347222222225
+SM37EFPD9ZVR3YRJE7673MJ3W0T350JM1HVZVCDC3,180555557
+SM28P04DXXDEY3WY02VQJD4TS7VZS00Z6SX54XH5J,356850000
+SM260QHD6ZM2KKPBKZB8PFE5XWP0MHSKTD1B7BHYR,2708333337
+SM1TP7CNY63KQY7DVRDVPNE1X73ND3PT8JX46971V,166667000000
+SM3TJ7J6DR9KFQXQGK9SY7M7AEJT33C8NV639J99C,42311000000
+-----END STX BALANCES-----
+-----BEGIN STX VESTING-----
+address,value,blocks
```

Comment:
For the test data, can you add some small-valued heights so there can be unit tests for token unlocking that don't require generating thousands of empty burnchain blocks?

Reply:
Related, should the stx-genesis lib use a debug/release (or unit test) conditional to toggle between including a small test file like this, vs the full production file?

Also, @kantai and I discussed including the (as-of-today) production chainstate.txt in this PR, because it does introduce some pain points that seem helpful to get accustomed to. @jcnelson thoughts?

Reply:
Let's go with @kantai's suggestion.

Reply:
Issue created https://github.com/blockstack/stacks-blockchain/issues/2137
Will do in a follow up PR